vod/media_set_parser.c

```diff
@@ -137,6 +146,13 @@ static json_object_key_def_t media_notification_params[] = {
 	{ vod_null_string, 0, 0 }
 };
 
+static json_object_key_def_t media_closed_captions_params[] = {
+	{ vod_string("id"), 							VOD_JSON_STRING, MEDIA_CLOSED_CAPTIONS_PARAM_ID },
+	{ vod_string("language"), 				VOD_JSON_STRING, MEDIA_CLOSED_CAPTIONS_PARAM_LANGUAGE },
+	{ vod_string("label"),						VOD_JSON_STRING, MEDIA_CLOSED_CAPTIONS_PARAM_LABEL },
+	{ vod_null_string, 0, 0 } 
```

Comment:
this should appear aligned with the rest of the tables when viewed with tab = 4 spaces